main.py

```diff
@@ -0,0 +1,75 @@
+import os
+
+from agent import get_graph
+from inputs import from_github
+from langchain_core.messages import HumanMessage
+from tools import get_pr_metadata
+
+from composio import Action
+
+import logging
+logging.getLogger().setLevel(logging.DEBUG)
+
+# TODO: 
+# 1. API_KEY
+# 2. Log
+# 3. RAG with langchain
+
+def main() -> None:
+    """Run the agent."""
```

Comment:
Add docstrings to the `main()` function and other parts of the code to explain their purpose and usage. Provide comments within the code to explain the steps being performed, especially in the `main.py` script.